bin/p2-shutdown/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/store/consul"
+	"github.com/square/p2/pkg/store/consul/flags"
+	"github.com/square/p2/pkg/types"
+)
+
+const helpMessage = `
+p2-shutdown is a command that is useful to gracefully shutdown pods on a host
+before doing maintenance. Ideally, these pods would be relocated to a different
+host but we live in a world where hosts are pets.
+`
+
+var (
+	verbose      = kingpin.Flag("verbose", "Print debugging information").Short('v').Bool()
+	dryRun       = kingpin.Flag("dry", "dry run").Short('d').Default("false").Bool()
+	shutdownPods = kingpin.Flag("pods", "The list of pods to shutdown. Leave empty for all").Short('p').Strings()
+	podRoot      = kingpin.Flag("podRoot", "The base directory for pods").Required().String()
```

Comment:
for convenience you could maybe default this to `pods.DefaultPath`. Also nit: `p2-run-hooks` calls it `--pod-root` instead of `--podRoot`